classes/pdf/HTMLTemplateInvoice.php

```diff
@@ -29,13 +29,20 @@
  */
 class HTMLTemplateInvoiceCore extends HTMLTemplate
 {
+    /**
+     * @var Order
+     */
     public $order;
+
+    /**
+     * @var OrderInvoice
+     */
     public $order_invoice;
-    public $available_in_your_account = false;
 
     /**
      * @param OrderInvoice $order_invoice
-     * @param $smarty
+     * @param Smarty $smarty
```

Comment:
The signature is `mixed` not `Smarty` even if it should be a `Smarty` instance :/

Reply:
It must be a Smarty instance to work

Reply:
So you can change the method signature too :wink: